integration/query_comparision_test.go

```diff
@@ -189,6 +189,14 @@ func TestQueryComparisonEq(t *testing.T) {
 			filter:      bson.D{{"value", bson.D{{"$eq", math.NaN()}}}},
 			expectedIDs: []any{"double-nan"},
 		},
+		"DoubleBigInt64": {
```

Comment:
Tests are grouped by argument type, not results. If the argument is `int64`, it should not be in this block

Reply:
added this to contributng guidelines.

Reply:
Fixed